src/Root.cpp

```diff
@@ -423,7 +423,20 @@ void cRoot::LoadWorlds(cSettingsRepositoryInterface & a_Settings, bool a_IsNewIn
 			continue;
 		}
 		FoundAdditionalWorlds = true;
-		cWorld * NewWorld = new cWorld(WorldName.c_str());
+		cWorld * NewWorld;
+		// Note that if there's a world.ini file present, the world will override the following worldTypes in cWorld::start
+		if (WorldName.find("_nether") != std::string::npos)  // If the world has the word nether, mark it as a nether world. (will be overridden by setting  if ini present)
```

Comment:
Do we have a project-specific convention for string comparison? Also, can I easily make this case-insensitive?